### ISSUE TITLE
Change naming of the model package

### ISSUE THREAD

**Issue by leewujung** [OSOceanAcoustics/echopype#135]
The `model` package performs calibration and depth bin+time bin-averages to produce MVBS, plus noise removal that has to be done before the averaging. However `model` is not a good name and is remnant of the early days when I was trying to use the model-view-controller design pattern. But this name is easily confused with physics-based scattering models.

Given what it does and that more analysis functions will be implemented in the `analyze` package, I think we can change `model` to `process`.

Along with the above discussion, @emiliom also suggested to simplify the importing to allow just `import echopype.EchoData`. With the name change, it should probably become `import echopype.Process`. The same could be done for the `convert` package so can do `import echopype.Convert as Convert` rather than `from echopype.convert import Convert`.

**Comment by emiliom** [comment]
To be clearer, using `Convert` as an example, what I'm thinking of is adding:
```python
from .convert import Convert
```
to [echopype/__init__.py](https://github.com/OSOceanAcoustics/echopype/blob/master/echopype/__init__.py).

Though I haven't tested any of this, I think that will enable these sort of less verbose access: 

```python
# 1
import ecopype
data = echopype.Convert('inputargs')

# 2
from echopype import Convert
data = Convert('inputargs')
```

**Comment by emiliom** [comment]
BTW, when you get around to renaming the `model` subpackage to `process`, make sure to do it using `git mv`, not the file system `mv`. That way the git history is preserved, otherwise you lose the commit history for each module.

**Comment by leewujung** [comment]
A note for when I get around to do this: 
`from .convert import Convert` was done in earlier version but then removed because I couldn't make it work with `sphinx-automodapi` to auto-generate API refs. Now that this sphinx extension seems to throw off readthedocs (remain to be fixed on PR #141 ) maybe it's time to switch to something else.

**Comment by leewujung** [comment]
This is closed by #156.
